lib/fathead/jquery/output.txt

```diff
@@ -1,1919 +1,1919 @@
-.add()	A			jQuery Traversing\njQuery Miscellaneous Traversal							<section class="prog__container"><p>Create a new jQuery object with elements added to the set of matched elements.</p><pre><code>$(selector).add(selector) → jQuery\n$(selector).add(elements) → jQuery\n$(selector).add(html) → jQuery\n$(selector).add(selection) → jQuery\n$(selector).add(selector, context) → jQuery</code></pre></section>	https://api.jquery.com/add/
-.addBack()	A			jQuery Traversing\njQuery Miscellaneous Traversal							<section class="prog__container"><p>Add the previous set of elements on the stack to the current set, optionally filtered by a selector.</p><pre><code>$(selector).addBack(selector) → jQuery</code></pre></section>	https://api.jquery.com/addBack/
+	R	jQuery()										
```

Comment:
Done. It was an interesting issue though. The parser adds redirects for each jQuery method by removing parenthesis, `jQuery` symbol and prepending `$` to the method name. However, when it comes the core jQuery function (which is named `jQuery`), dollarizing it and removing `jQuery` symbol creates an empty title. 
Fixed it by applying a simple filter to check for alphanumerics in title  🎉